.docker/intelmq-full-dev/entrypoint_dev.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+/opt/dev/update.sh
+
+if [ "${LOG_MAIL_ENABLED}" = "true" ]; then
+    sed -i "s/return\ logger/### Code added to fix unexistent mail handler ###\n    mail_handler=logging.handlers.SMTPHandler(mailhost = ('${LOG_MAIL_MAILHOST}', ${LOG_MAIL_PORT}),fromaddr = '${LOG_MAIL_FROMADDR}',toaddrs = ['${LOG_MAIL_TOADDR}'],subject = '${LOG_MAIL_SUBJECT}',credentials = ${LOG_MAIL_CREDENTIALS}, secure = ${LOG_MAIL_SECURE} )\n    mail_handler.setLevel(${LOG_MAIL_LEVEL})\n    mail_handler.setFormatter(logging.Formatter(LOG_FORMAT))\n    logger.addHandler(mail_handler)\n    aux_logger = logger\n    return aux_logger\n    ### End code added to fix unexistent mail handler ###/g" /opt/intelmq/intelmq/lib/utils.py
+fi
+
+if [ "${ENABLE_BOTNET_AT_BOOT}" = "true" ]; then
+	intelmqctl start
+fi
+
+
+/opt/entrypoint.sh
```

Comment:
Please make sure to `export INTELMQ_IS_DOCKER=1` in your entrypoint.sh as there are some docker-specific handlers inside intelmq.

https://github.com/certat/intelmq-docker/blob/main/entrypoint.sh

Reply:
I think it is not necessary, because on line 14 we call the original entrypoint